grammars/SeedPython.g4

```diff
@@ -48,6 +48,10 @@
 
```

Comment:
Not related to this PR: it seems there is a trailing space at line #15. Please clear it. You may want to install a VS Code extension called "trailing Spaces" to check them.

Reply:
Done

Reply:
Done